### PR TITLE
Tech debt2021

### PR DESCRIPTION
**Pull request by Rhianna20**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ references:
   container_config_node: &container_config_node
     working_directory: ~/project
     docker:
-      - image: circleci/node:8-browsers
+      - image: circleci/node:14
```
Comment (wheresrhys) [inline .circleci/config.yml:14]: This _might_ still need `-browsers` on the end. @i-like-robots would you know?

**Comment by GeoffThorpeFT** [comment]
I think you also need to run `make verify` to reformat the code as a couple of the lines are triggering a circleci test failure: https://app.circleci.com/pipelines/github/Financial-Times/athloi/316/workflows/0a049366-f3bd-4103-ab62-c343e3a682d0/jobs/1959

**Review by GeoffThorpeFT — APPROVED** [review]
Many thanks for those final fixes.

File: .circleci/config.yml (hunk 1 above)
Comment (i-like-robots) [reply .circleci/config.yml]: I don't think so, it's got a very straightforward test setup 🤔

**Pull request merged**